### PR TITLE
Updated insights-partnerships.md

### PR DESCRIPTION
**Pull request by smamindla57**
Hii CircleCi dev team,

We have many users on our platform who are using CircleCi and would like to instrument it. We’d love to provide a smooth experience for CircleCi users who need to use monitoring solutions, such as New Relic. Please see our [documentation here](https://newrelic.com/instant-observability/circleci).

Please consider adding a link for your direct users to provide an option for New Relic.  For your convenience, we have submitted this PR with our proposed content.

Please feel free to let us know if you have any questions or concerns.

Thank you for your consideration.

### CODE REVIEW

**Comment by devinmarieb** [comment]
@smamindla57 Thanks, we'll check it out!

File: jekyll/_cci2/insights-partnerships.md [hunk 1]
```diff
@@ -13,7 +13,43 @@ version:
...
+
+**Step 4.** In the sidebar of your Project Settings, click **Webhooks**.
+
+**Step 5.** Click **Add webhook**.
+
+**Step 6.** Name your webhook.
+
+**Step 7.** Enter New Relic's **Logs endpoint:**
+
+US:
```
Comment (corinnesollows) [inline jekyll/_cci2/insights-partnerships.md:38]: Is this US and Canada?

**Review by corinnesollows — CHANGES_REQUESTED** [review]

File: jekyll/_cci2/insights-partnerships.md (hunk 1 above)
Comment (smamindla57) [reply jekyll/_cci2/insights-partnerships.md:38]: @corinnesollows, While creating a new account at New Relic, a user can choose their preferred data storage region i.e., US or Europe. Please note that the data center chosen doesn’t have to match with the physical location of the user.

**Review by kbatuigas — APPROVED** [review]
Thank you!

File: jekyll/_cci2/insights-partnerships.md (hunk 1 above)
Comment (kbatuigas) [reply jekyll/_cci2/insights-partnerships.md:38]: @corinnesollows This sounds good to me, do we need any further changes or are we OK to go ahead and merge?

Comment (smamindla57) [reply jekyll/_cci2/insights-partnerships.md:38]: @kbatuigas, everything is good, please go ahead! 
Thank you.

**Review by corinnesollows — APPROVED** [review]

**Pull request merged**